kitsune/lib/email.py

```diff
@@ -0,0 +1,66 @@
+import logging
+
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+log = logging.getLogger('k.lib.email')
+RealBackend = import_string(settings.EMAIL_LOGGING_REAL_BACKEND)
+
+
+class LoggingEmailBackend(RealBackend):
```

Comment:
Exactly. I filed [bug 1223471](https://bugzilla.mozilla.org/show_bug.cgi?id=1223471) for the `settings_local.py` change.